content/ru/event-streaming.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Потоковая передача событий
+status: Completed
+category: concept
+tags: ["methodology", "networking", ""]
+---
+
+Потоковая передача событий — это подход, при котором одно приложение отправляет данные о событиях в другие, чтобы постоянно сообщать им о том, что оно делает.
+Представьте себе службу, транслирующую все, что она делает, всем остальным службам.
+Каждое действие, выполняемое такой службой, называется событием, отсюда и потоковая передача событий.
+Например, Служба автоматизированных котировок Национальной ассоциации дилеров по ценным бумагам (NASDAQ) каждую секунду публикует свежую информацию о ценах на акции и сырьё.
```

Comment:
I'm not sure whether we need to have all this long transcription in Russian here :thinking:

Reply:
I think it's necessary. For a Russian-speaking reader, the meaning of the term may not be clear.